cmd/setec/setec.go

```diff
@@ -140,6 +145,10 @@ var serverArgs struct {
 	Dev        bool   `flag:"dev,Run in developer mode"`
 }
 
+var clientArgs struct {
+	Server string `flag:"s,default=$SETEC_SERVER,Server address"`
```

Comment:
Is the default thing new behavior in flax, or something that was already there?

Reply:
Both. Defaults were already there, and I later added support for environment variables.